### PR TITLE
Create cerebras.yaml

### PR DESCRIPTION
**Pull request by ronitmndl**
1. Adds [Cerebras-GPT](https://www.cerebras.net/blog/cerebras-gpt-a-family-of-open-compute-efficient-large-language-models/) model (ran precommit.sh)
2. Will make necessary changes when the arXiv paper is published

### CODE REVIEW

File: assets/cerebras.yaml [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+  created_date:
+    value: 2023-03-28
+    explanation: >
+      The date of the blog post announcing Cerebras-GPT
+      [[Cerebras Blog Post]]
+      (https://www.cerebras.net/blog/cerebras-gpt-a-family-of-open-compute-efficient-large-language-models).
+  url: https://www.cerebras.net/blog/cerebras-gpt-a-family-of-open-compute-efficient-large-language-models/
+  model_card: https://huggingface.co/cerebras/Cerebras-GPT-13B
+  modality: text (English)
+  size: 13B parameters (dense model)
```
Comment (rishibommasani) [inline assets/cerebras.yaml:22]: Could you include in the explanation that this is the size of the largest model?

**Review by rishibommasani — APPROVED** [review]
Fantastic, looks great to me! For the future, feel free to assign me as a reviewer!

**Review by rishibommasani — APPROVED** [review]

**Comment by ronitmndl** [comment]
Made an error in the formatting of the model  size explanation. Corrected in #119 pull request.

**Comment by ronitmndl** [comment]
> Fantastic, looks great to me! For the future, feel free to assign me as a reviewer!

The option to assign a reviewer is disabled for me. I think you need write access to do that.

**Pull request merged**